config.py

```diff
@@ -11,28 +12,33 @@
 param["model_regions"] = 'NUTS0_wo_Balkans'
 param["year"] = 2015
 
-load = {"dict_season": {1:'Winter', 2:'Winter', 3:'Spring/Fall', 4:'Spring/Fall', 5:'Spring/Fall', 6:'Summer',
-                        7:'Summer', 8:'Summer', 9:'Spring/Fall', 10:'Spring/Fall', 11:'Spring/Fall', 12:'Winter'},
+load = {"dict_season": {1: 'Winter', 2: 'Winter', 3: 'Spring/Fall', 4: 'Spring/Fall', 5: 'Spring/Fall', 6: 'Summer',
+                        7: 'Summer', 8: 'Summer', 9: 'Spring/Fall', 10: 'Spring/Fall', 11: 'Spring/Fall', 12: 'Winter'},
         "dict_daytype": {'Monday': 'Working day', 'Tuesday': 'Working day', 'Wednesday': 'Working day',
-                         'Thursday': 'Working day', 'Friday': 'Working day', 'Saturday':'Saturday', 'Sunday':'Sunday'},
-		# If the data source of the load time series uses different country codes
-        "dict_countries": {'AT': 'AT', 'BA': 'BA', 'BE': 'BE', 'BG': 'BG', 'CH': 'CH', 'CY': 'CY', 'CZ': 'CZ', 'DE': 'DE',
-		                   'DK': 'DK', 'EE': 'EE', 'ES': 'ES', 'FI': 'FI', 'FR': 'FR', 'GB': 'UK', 'GR': 'EL', 'HR': 'HR',
-                           'HU': 'HU', 'IE': 'IE', 'IS': 'IS', 'IT': 'IT', 'LT': 'LT', 'LU': 'LU', 'LV': 'LV', 'ME': 'ME',
-                           'MK': 'MK', 'NI': 'UK', 'NL': 'NL', 'NO': 'NO', 'PL': 'PL', 'PT': 'PT', 'RO': 'RO', 'RS': 'RS',
+                         'Thursday': 'Working day', 'Friday': 'Working day', 'Saturday': 'Saturday',
+                         'Sunday': 'Sunday'},
+        # If the data source of the load time series uses different country codes
+        "dict_countries": {'AT': 'AT', 'BA': 'BA', 'BE': 'BE', 'BG': 'BG', 'CH': 'CH', 'CY': 'CY', 'CZ': 'CZ',
+                           'DE': 'DE',
+                           'DK': 'DK', 'EE': 'EE', 'ES': 'ES', 'FI': 'FI', 'FR': 'FR', 'GB': 'UK', 'GR': 'EL',
+                           'HR': 'HR',
+                           'HU': 'HU', 'IE': 'IE', 'IS': 'IS', 'IT': 'IT', 'LT': 'LT', 'LU': 'LU', 'LV': 'LV',
+                           'ME': 'ME',
+                           'MK': 'MK', 'NI': 'UK', 'NL': 'NL', 'NO': 'NO', 'PL': 'PL', 'PT': 'PT', 'RO': 'RO',
+                           'RS': 'RS',
                            'SE': 'SE', 'SI': 'SI', 'SK': 'SK'},
         # Replacement of missing countries with profile (replacement) and annual values
-		"replacement": 'MK',
-		"missing_countries": {'AL': 6376000, 'KS': 2887000}, # in MWh
+        "replacement": 'MK',
+        "missing_countries": {'AL': 6376000, 'KS': 2887000, 'UK': 6376000, 'EL': 2887000},  # in MWh
```

Comment:
why add UK and EL here?

Reply:
They are not present in the ENTSOE Database

Reply:
but the total load is wrong

Reply:
Yes, I forgot to set a proper value. It was just a place holder.